app/assets/stylesheets/partials/_table_tree_view.scss

```diff
@@ -63,3 +63,13 @@ $table-tree-view-gutter: 1.5em;
     @include govuk-shape-arrow($direction: down, $base: 14px);
   }
 }
+
+
+table.govuk-table {
+  display: block;
+  overflow-x: scroll;
+}
+
+.govuk-template {
+    overflow-x:hidden;
+}
```

Comment:
Could we have this in a different partial? (called _table.scss or similar). This partial refers to the tree view of activities (with the drop down list of activites), so there's a degree of misdirection here